import-export-cli/cmd/mg/undeployAPI.go

```diff
@@ -41,10 +45,14 @@ const (
 
 var undeployAPICmdExamples = utils.ProjectName + ` ` + mgCmdLiteral + ` ` + undeployCmdLiteral + ` ` + apiCmdLiteral + ` --environment dev -n petstore -v 0.0.1
    ` + utils.ProjectName + ` ` + mgCmdLiteral + ` ` + undeployCmdLiteral + ` ` + apiCmdLiteral + ` -n petstore -v 0.0.1 -e dev --vhost www.pets.com 
-   ` + utils.ProjectName + ` ` + mgCmdLiteral + ` ` + undeployCmdLiteral + ` ` + apiCmdLiteral + ` -n SwaggerPetstore -v 0.0.1 --environment dev` +
+   ` + utils.ProjectName + ` ` + mgCmdLiteral + ` ` + undeployCmdLiteral + ` ` + apiCmdLiteral + ` -n petstore -v 0.0.1 -e dev --gateway-env "Production and Sandbox" Label1 Label2
+   ` + utils.ProjectName + ` ` + mgCmdLiteral + ` ` + undeployCmdLiteral + ` ` + apiCmdLiteral + ` -n petstore -v 0.0.1 -e dev --vhost www.pets.com --gateway-env "Production and Sandbox" 
+   ` + utils.ProjectName + ` ` + mgCmdLiteral + ` ` + undeployCmdLiteral + ` ` + apiCmdLiteral + ` -n SwaggerPetstore -v 0.0.1 --environment dev
 
-	"\n\nNote: The flags --name (-n), --version (-v), --environment (-e) are mandatory. " +
-	"The user needs to be logged in to use this command."
+Note: The flags --name (-n), --version (-v), --environment (-e) are mandatory.
+If the flag (--gateway-env (-g)) is not provided, API will be undeployed from all deployed gateway environments.
```

Comment:
When the -g flag is not given we undeploy from all the gateway environments?

Reply:
yes, if -g not provided we are going to undeploy from all envs, this is the same behaviour of synapse gw command `apictl undeploy api`, @Rajith90 @pubudu538  @1akshitha can u confirm?

Reply:
Does synapse deploy an API to all gateway envs when -g is not provided too?

Reply:
yes. That's the implementation we used with synapse. Basically the command itself explains that we are undeploying the API. With that command, we should undeploy from all the gateways. For that i don't think we need another flag. IMO we should minimise the number of flags we use because that'll be less code and easier to understand.

Command use case would be, if someone needs to undeploy the api (this means we should not keep the api in any of the gateways), use apictl undeploy api. If the user needs to undeploy from specific gateways only, then provide them through the -g flag.

Reply:
There is no command `deploy` actually, the deployment is done via `apictl import` command and in APIM side it checks for a file `deployment_environments.yaml`, deployment is done to environments defined in this file. @1akshitha this is the behaviour, right?

Reply:
yes, That's the behaviour. Since that'll complicate the revision process, we decided not to implement that and provide the same functionality with the import command.

Reply:
The use case that troubles me is the following. Since the synapse gw doesn't have a deploy command it doesn't apply there I guess.

user 1: doesn't give -g when deploying and therefore it goes to the dafault env "Production and Sandbox"
user 2: gives an env name "Region2" and deploys the same API
user 1: doesn't know that a "Region2" exists and is being used. Wants to undeploy the API just like it was deployed, and uses undeploy without -d.

"Region2" also gets lost.

Reply:
We have not supported the flag `-g` for `apictl mg deploy` command, instead we read the file `deployment_environments.yaml` and find the environments the API to be deployed (currently I am working on this change).